src/pages/index.tsx

```diff
@@ -1,17 +1,93 @@
 import React, { FunctionComponent } from "react";
 import Head from "next/head";
+import Aside from "../components/Aside";
+import { StartButton } from "../components/Button";
 
-const Home: FunctionComponent = () => {
-  return (
-    <>
-      <Head>
-        <title>Beacon Registration Service - Register a new beacon</title>
-      </Head>
-      <h1 className="govuk-heading-l">Header goes here</h1>
+const ServiceStartPage: FunctionComponent = () => (
+  <>
+    <Head>
+      <title>
+        Beacon Registration Service - Register a new 406 MHz distress beacon
+      </title>
+    </Head>
 
-      <p className="govuk-body">Service start page to go here!</p>
-    </>
-  );
-};
+    <Header />
 
-export default Home;
+    <Body />
```

Comment:
I really like the way you've abstracted away the raw HTML 👍🏻   

Could the semantic content be retained up here in the `<ServiceStartPage>` component?  Currently, the top view of every `page.tsx` will look near-identical with just `<Header/Heading/PageHeading />` and `<Body />` tags, which I think will increase the cognitive burden when we're working on a service flow between several pages?

What do you think about something like this in the `<ServiceStartPage>` component?

```jsx
    <MainContent>
        <PageHeading>Register a UK 406 Mhz beacon</PageHeading>

        <AboutTheService />

        <StartButton />

        <OtherWaysToAccessTheService/>    
    </MainContent>
    
    <Aside>
        <RelatedContent />
    </Aside>
```